ntc_rosetta/parsers/openconfig/ios/openconfig_interfaces/interfaces.py

```diff
@@ -93,7 +93,7 @@ class Yangify(ParserData):
         path = "/openconfig-interfaces:interfaces/interface"
 
         def extract_elements(self) -> Iterator[Tuple[str, Dict[str, Any]]]:
-            for k, v in self.native["interface"].items():
+            for k, v in jh.query("interface", self.native, default={}).items():
```

Comment:
Is this to prevent a key error? I believe I saw that previously.

Reply:
Correct.  I keep finding little things about YANG internals.  Many of these attributes aren't required, so we need to pass over them if they don't exist instead of raising an error.  This will ultimately save us a headache with merge behavior.